apps/explorer/test/support/fakes/voter_rewards_for_group.ex

```diff
@@ -33,15 +42,12 @@ defmodule Explorer.Fakes.VoterRewardsForGroup do
     byte_count: 32,
     bytes: <<0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 5>>
   }
-  @block_6_number 10_782_720
-  @block_6_hash %Explorer.Chain.Hash{
-    byte_count: 32,
-    bytes: <<0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 6>>
-  }
 
-  def calculate(voter_address_hash, group_address_hash, to_date \\ DateTime.utc_now())
+  def calculate(voter_address_hash, group_address_hash, _to_date \\ DateTime.utc_now())
```

Comment:
It doesn't really make sense to have a default argument for something you're ignoring